src/dataExplorer/components/OpenScript.tsx

```diff
@@ -41,11 +42,11 @@ const OpenScript: FC<Props> = ({onCancel, onClose}) => {
   const history = useHistory()
   const org = useSelector(getOrg)
 
-  const handleGetScripts = useCallback(async () => {
+  const handleGetScripts = useCallback(async (name: string = '') => {
     try {
       if (getScripts) {
         setLoading(RemoteDataState.Loading)
-        const resp = await getScripts({})
+        const resp = await getScripts({query: {limit: 250, name}})
```

Comment:
Not really anything actionable here, necessarily, but if we're going to lean on search, even 250 might be too much? Maybe 100? 50? Just throwing ideas out there. No need to change unless you feel like it.